src/main/resources/static/app/request/request-contents.html

```diff
@@ -347,6 +347,20 @@ <h4 class="h4">{{ 'Letter of medical research ethics committee' | translate }}</
     </div>
 
     <div ng-if="isPalga()">
+
+        <!-- Request type -->
+        <h4 class="h4 h4-palga">{{'Request type'|translate}}</h4>
+        <hr>
```

Comment:
The `<hr>` is only here, which makes it inconsistent

Reply:
oups, copy-paste from request-review...